lib/asn1/decoders/der.js

```diff
@@ -159,8 +159,9 @@ DERNode.prototype._decodeObjid = function decodeObjid(buffer, values, relative)
   else
     result = [first, second].concat(identifiers.slice(1));
 
-  if (values)
-    result = values[result.join(' ')];
+  var key = result.join(' ');
+  if (values && typeof values[key] !== 'undefined')
```

Comment:
Is there any point in using `typeof` now? You can't override `undefined` anymore :)